examples/go-tracepoint-counter/main.go

```diff
@@ -136,7 +148,7 @@ func loadProgram(paramData *configMgmt.ParameterData) (func(string), error) {
 	c := gobpfd.NewLoaderClient(conn)
 	loadRequestCommon := &gobpfd.LoadRequestCommon{
 		Location:    paramData.BytecodeSource.Location,
-		SectionName: "stats",
+		SectionName: "tracepoint_kill_recorder",
```

Comment:
Wow this _shouldn't_ have been working since passed SectionName didn't match the one on the image... Hrm 

https://github.com/bpfd-dev/bpfd/blob/main/bpfd/src/command.rs#L244

We definitely need and integration test for this :) 

I wonder how it was

Reply:
I think it was only really tested on K8s. Don't think it ever worked when run locally.

Reply:
Ahhh duh you're right, thanks!